test/e2e_node/cpu_manager_test.go

```diff
@@ -1172,6 +1302,83 @@ func isSMTAlignmentError(pod *v1.Pod) bool {
 	return re.MatchString(pod.Status.Reason)
 }
 
+// getNumaNodeCPUs retrieves CPUs for each NUMA node.
+func getNumaNodeCPUs() (map[int][]int, error) {
```

Comment:
if you return cpuset.CPUSet, which we need to compute anyway...

Reply:
Update, Thanks!